4-projects/common.auto.tfvars

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
Thanks @renato-rudnicki  for the pull request.

The file `4-projects/common.auto.tfvars` does not need to be added, it will be created after following this instruction in the README:

https://github.com/terraform-google-modules/terraform-example-foundation/blob/a3a413d623d7e5a312dcfa84a67e3ff528e14abf/4-projects/README.md#L25